trax/rl/actor_critic.py

```diff
@@ -83,6 +84,9 @@ def __init__(self, task,
     value_output_dir = kwargs.get('output_dir', None)
     if value_output_dir is not None:
       value_output_dir = os.path.join(value_output_dir, '/value')
+      # If needed, create value_output_dir and missing parent directories.
+      if not tf.io.gfile.isdir(value_output_dir):
```

Comment:
This is not needed (looks like supervised trainer is going to create this directory anyway).
But you need https://github.com/google/trax/pull/397  . Otherwise it will try to write in dir  `/value` (in root dir)